contrib/platform/test/com/sun/jna/platform/win32/Wtsapi32Test.java

```diff
@@ -100,4 +103,59 @@ public void testWTSEnumerateProcessesEx() {
                 Wtsapi32.INSTANCE.WTSFreeMemoryEx(Wtsapi32.WTS_PROCESS_INFO_LEVEL_1, pProcessInfo, pCount.getValue()));
 
     }
+
+    public void testWTSEnumerateSessions() {
+        int WTSClientAddress = 14;
+        int WTSSessionInfo = 24;
+        int WTSClientProtocolType = 16;
```

Comment:
Would it make sense to move this to `Wtsapi32.java` as 
```
public interface _WTS_INFO_CLASS {
    int WTSClientAddress = 14;
    int WTSClientProtocolType = 16;
    int WTSSessionInfo = 24;
}
```
?

Reply:
Sure... although I'd probably map the entire enum there.  I was being a bit lazy...